api/v1alpha2/types.go

```diff
@@ -111,20 +111,17 @@ type VnetSpec struct {
 	CidrBlock string `json:"cidrBlock,omitempty"`
 
 	// Tags is a collection of tags describing the resource.
-	// TODO: Uncomment once tagging is implemented.
-	//Tags tags.Map `json:"tags,omitempty"`
+	Tags Tags `json:"tags,omitempty"`
 }
 
-// TODO: Implement tagging
-/*
-// Tags defines resource tags.
-type Tags map[string]*string
-*/
+// IsUnmanaged returns true if the vnet is unmanaged.
+func (v *VnetSpec) IsUnmanaged(clusterName string) bool {
```

Comment:
yeah I tend to agree with you, I used `IsUnmanaged` because that's how the aws provider named it for consistency... But now that there's 2 of us thinking `IsManaged` is better, I will change it :)

Reply:
Count me as well, "positive" names avoid double-negatives, which are harder to process